tests/integration/test_logcollector/test_command_monitoring/test_command_execution_freq.py

```diff
@@ -56,60 +56,70 @@
     - logcollector_cmd_exec
 '''
 import os
+import sys
+import subprocess
 import pytest
 from datetime import timedelta, datetime
 
-from wazuh_testing import global_parameters, logger
+from wazuh_testing import global_parameters, logger, T_30, T_60
 from wazuh_testing.tools.time import TimeMachine
-import wazuh_testing.logcollector as logcollector
-from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.modules.logcollector import LOG_COLLECTOR_PREFIX, WINDOWS_AGENT_PREFIX, \
+                                               GENERIC_CALLBACK_ERROR_COMMAND_MONITORING
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.modules.logcollector import event_monitor as evm
+
 
 # Marks
 pytestmark = [pytest.mark.tier(level=0)]
 
-# Configuration
-test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-configurations_path = os.path.join(test_data_path, 'wazuh_command_conf.yaml')
+prefix = LOG_COLLECTOR_PREFIX
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# ------------------------------------------------ TEST_ACCEPTED_VALUES ------------------------------------------------
+# Configuration and cases data
+t1_configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_execution_freq.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_execution_freq.yaml')
+
+# Accepted values test configurations (t1)
+t1_configuration_parameters, t1_configuration_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+t1_configurations = load_configuration_template(t1_configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
 
 local_internal_options = {'logcollector.remote_commands': '1', 'logcollector.debug': '2', 'monitord.rotate_log': '0',
                           'windows.debug': '2'}
 
+if sys.platform == 'win32':
+    location = r'C:\testing\file.txt'
+    prefix = WINDOWS_AGENT_PREFIX
+
+
+@pytest.fixture(scope='module')
```

Comment:
Move to logcollector module conftest

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3486/commits/21f9ee85c4d9203435d6adca8336956810f20708)